scene/2d/canvas_item.cpp

```diff
@@ -784,22 +784,22 @@ void CanvasItem::draw_rect(const Rect2 &p_rect, const Color &p_color, bool p_fil
 				p_antialiased);
 		VisualServer::get_singleton()->canvas_item_add_line(
 				canvas_item,
-				p_rect.position + Point2(0, offset),
-				p_rect.position + Size2(0, p_rect.size.height - offset),
+				p_rect.position + Point2(p_rect.size.width, offset),
+				p_rect.position + Size2(p_rect.size.width, p_rect.size.height - offset),
```

Comment:
The order used to be "add `Point2` then add `Size2`, but now it's no longer consistent, you have `Point2`/`Size2` here, but `Size2`/`Point2` for the next two calls.
It's the same typedef so it doesn't matter so much, but the inconsistency looks weird.

Reply:
Yes. Size2 for all probably makes more sense semantically.